src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -644,14 +646,14 @@ public partial class UdpClient : System.IDisposable
         public void JoinMulticastGroup(System.Net.IPAddress multicastAddr) { }
         public void JoinMulticastGroup(System.Net.IPAddress multicastAddr, int timeToLive) { }
         public void JoinMulticastGroup(System.Net.IPAddress multicastAddr, System.Net.IPAddress localAddress) { }
-        public byte[] Receive(ref System.Net.IPEndPoint remoteEP) { throw null; }
+        public byte[] Receive(ref System.Net.IPEndPoint? remoteEP) { throw null; }
```

Comment:
Should `remoteEP` here have a `[NotNull]` attribute on it? i.e. we allow it to be null when passed in, but it'll never be null on the way out?

Reply:
Good call. Added.